### ISSUE TITLE
M30T thermal region returns an error

### ISSUE THREAD

**Issue by eric-dk** [dji-sdk/Mobile-SDK-Android-V5#18]
Steps-to-reproduce:
With the correct `ComponentIndexType` and `CameraLensType.CAMERA_LENS_THERMAL` use `CameraKey.KeyThermalRegionMetersureArea.set(…)` with `DoublePoint2D` where *x*, *y*, *width*, *height* are 0…1.

Expected:
Camera restricts thermal measuring region.

Actual:
`CameraKey.KeyThermalAreaTemperatureAggregations.listen(…)` still returns points outside of the specified region.
```
DJIError{errorCode='CORE_INVALID_PARAMETERS', innerCode='', description='null', hint='error code = -483'}
```

**Comment by DJI-William** [comment]
First of all you should call KeyThermalTemperatureMeasureMode to set the thermal measurement mode as REGION. Then call KeyThermalRegionMetersureArea to set the region. Lastly, listen the KeyThermalRegionMetersureTemperature for returned result.
![image](https://user-images.githubusercontent.com/77274755/179167730-4436a4a5-3e7a-426a-96d6-11c9f1005a09.png)

**Comment by eric-dk** [comment]
All prerequisites done; same error.

**Comment by DJI-William** [comment]
Please have a look my testing video and it works. But I found the key is different witht the api doc, I will discuss with the product manager.
Link：https://pan-sec.djicorp.com/s/QDkHTQt85f5tdnE
dji123

**Comment by DJI-William** [comment]
Sorry my bad, I was using Beta3 demo, the official 5.0.0 is released and the key is rename to KeyThermalRegionMetersureTemperature and AREAt to REGION. It should work.

**Comment by DJI-William** [comment]
@eric-dk is the problem solved in 5.0 version?
